sql/testdata/timedate

```diff
@@ -0,0 +1,38 @@
+statement ok
```

Comment:
Minor point, but I usually see date & time operations referred to as `datetime`, not `timedate`.